WordPress/Classes/ViewRelated/Plans/PlanPostPurchaseViewController.swift

```diff
@@ -335,9 +335,11 @@ enum PlanPostPurchasePageType: Int {
     case Themes
     
     static func pageTypesForPlan(plan: Plan) -> [PlanPostPurchasePageType] {
-        switch plan.slug {
-        case "premium": return [.PurchaseComplete, .Customize, .VideoPress]
-        case "business": return [.PurchaseComplete, .Customize, .VideoPress, .Themes]
+        switch plan.id {
+        // Premium
+        case 1003: return [.PurchaseComplete, .Customize, .VideoPress]
```

Comment:
Makes sense. Will the post purchase screens stay hardcoded, or we'll be fetching that info from the API as well?